### PR TITLE
updated CLI docs to reflect new name now available for Windows

### PR DESCRIPTION
**Pull request by taiyosogawa**
When following the install instructions, the Windows PS script will install the latest bundle, which has already removed vso. When the Mac and Linux install methods have been updated, I will fully replace all "vso" with "codespaces".

I also cleaned some places where we use "environments" instead of "Codespaces".

### CODE REVIEW

File: vscodespaces/TOC.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@
   - name: Repository support
     href: reference/repository.md
   - name: Codespaces CLI
-    href: reference/vsonline-cli.md
+    href: reference/codespaces-cli.md
```
Comment (abuchholtzau) [inline vscodespaces/TOC.yml:39]: Can you undo this? There are a couple more steps required for file renames so I'm going to deal with them in bulk!

File: vscodespaces/how-to/self-hosting-cli.md [hunk 2]
```diff
@@ -3,44 +3,44 @@ author: abuchholtzau
...
-You can host a Codespaces environment on your own hardware using the headless CLI. This article describes how to register and connect to a headless Codespaces server.
+You can host a Codespace on your own hardware using the headless CLI. This article describes how to register and connect to a headless Codespaces server.
 
 > [!NOTE]
 > A Microsoft Account and Azure Subscription are required to use Visual Studio Codespaces. Sign up for a Microsoft Account and Azure Subscription at [https://azure.microsoft.com/free/](https://azure.microsoft.com/free/).
 
 ## Install Codespaces for self-hosting
 
-Codespaces CLI tools can be used on [Windows](~/reference/vsonline-cli.md#windows), [macOS](~/reference/vsonline-cli.md#macos), and [Linux](~/reference/vsonline-cli.md#linux) environments. For installation instructions, see [Codespaces CLI Reference](~/reference/vsonline-cli.md#installation).
+Codespaces CLI tools can be used on [Windows](~/reference/codespaces-cli.md#windows), [macOS](~/reference/codespaces-cli.md#macos), and [Linux](~/reference/codespaces-cli.md#linux) environments. For installation instructions, see [Codespaces CLI Reference](~/reference/codespaces-cli.md#installation).
```
Comment (abuchholtzau) [inline vscodespaces/how-to/self-hosting-cli.md:20]: Undo given file rename will happen later!

File: vscodespaces/how-to/self-hosting-cli.md [hunk 3]
```diff
@@ -3,44 +3,44 @@ author: abuchholtzau
...
 
 ## Register your machine
 
 You must create a Codespaces plan before starting the registration process. For more information on creating a plan, see [create a plan](browser.md#create-a-plan) in our browser how-to guide.
 
-Once you've successfully installed the CLI, start the interactive registration process by running the [`vso start`](~/reference/vsonline-cli.md#start-codespaces) command.
+Once you've successfully installed the CLI, start the interactive registration process by running the [`vso start`](~/reference/codespaces-cli.md#start-codespaces) command.
```
Comment (abuchholtzau) [inline vscodespaces/how-to/self-hosting-cli.md:26]: Undo file rename.

**Comment by abuchholtzau** [comment]
Or alternatively, you can feel free to do the rename but happy to batch them!
The process:

•	Rename the file
•	Update any references (build process usually catches them and warns if you miss one)
•	Create a redirect entry
Redirects are added to the file .openpublishing.redirection.json which lives at root in the repo. They are of the format:

•	source_path: the path of the new renamed file
•	redirect_url: the live [old] URL that should be redirected
•	redirect_document_id: I don't know what this does, I always leave it false 😋

File: vscodespaces/TOC.yml (hunk 1 above)
Comment (taiyosogawa) [reply vscodespaces/TOC.yml]: No problem - done!

File: vscodespaces/how-to/self-hosting-cli.md (hunk 2 above)
Comment (taiyosogawa) [reply vscodespaces/how-to/self-hosting-cli.md]: Done!

File: vscodespaces/reference/vsonline-cli.md [hunk 4]
```diff
@@ -132,26 +138,32 @@ The following **commands** are available:
 
 The **`start`** command begins the interactive process of registering your self-hosted machine:
   
+  Windows/Mac:
+  ```bash
+  vso start [args]
```
Comment (edgonmsft) [inline vscodespaces/reference/vsonline-cli.md:143]: codespaces?

File: vscodespaces/reference/vsonline-cli.md [hunk 5]
```diff
@@ -132,26 +138,32 @@ The following **commands** are available:
...
+  ```
+
+  Linux:
   ```bash
   vso start [args]
```
Comment (edgonmsft) [inline vscodespaces/reference/vsonline-cli.md:148]: Are we not changing this one yet?

File: vscodespaces/reference/vsonline-cli.md [hunk 6]
```diff
@@ -167,8 +179,13 @@ Show help information
...
-  
+The **`stop`** command removes your Codespace from the Visual Studio Codespaces service:
+
+  Windows/Mac:
+  ```bash
+  codespaces stop
+  ```  
+  Linux:
   ```bash
   vso stop
```
Comment (edgonmsft) [inline vscodespaces/reference/vsonline-cli.md:190]: And this one.

File: vscodespaces/reference/vsonline-cli.md (hunk 5 above)
Comment (taiyosogawa) [reply vscodespaces/reference/vsonline-cli.md:148]: we will soon! I will update :)

File: vscodespaces/reference/vsonline-cli.md (hunk 6 above)
Comment (taiyosogawa) [reply vscodespaces/reference/vsonline-cli.md:190]: will update when it's ready :)

File: vscodespaces/reference/vsonline-cli.md (hunk 4 above)
Comment (taiyosogawa) [reply vscodespaces/reference/vsonline-cli.md]: good catch - done!

**Pull request merged**